Tasks/PowerShellOnTargetMachines/PowerShellOnTargetMachines.ps1

```diff
@@ -110,6 +135,9 @@ if($runPowershellInParallel -eq "false" -or  ( $resources.Count -eq 1 ) )
         {
             Write-Telemetry "DTLSDK_Error" $deploymentResponse.DeploymentSummary
             Write-Verbose $deploymentResponse.Error.ToString()
+            
+            Write-DTLServiceDeprecationMessageIfRequired $machine
```

Comment:
This is already being done at the end of the script. Why do you need to do it here as well?

Reply:
this is for the flow in case of seq flow and the other end condition is for parallel scenario